getting-started/index.md

```diff
@@ -18,6 +18,19 @@ top = false
 Your topside computer’s network configuration should be the same as for the previous Companion software.
 To configure it, you can follow our [network setup instructions](https://www.ardusub.com/quick-start/installing-companion.html#network-setup).
 
+## USB OTG
+It's also possible to connect with the Raspberry though the USB-C connection. Once it's connected the system should be available though `blueos.local`. Since the `usb0`interface by default has DHCP server enable, it's not possible to access it with `192.168.2.2` address.
```

Comment:
Just grammar:
```suggestion
It's also possible to connect to the Raspberry through its USB-C connection. Once it's connected, the system should be available at `blueos.local`. Since the `usb0` interface has a DHCP server enabled by default, it's not possible to access it with the `192.168.2.2` address.
```

About the text, it's not clear why `192.168.2.2` (the default BlueOS static IP address) would not work anymore. Might be better to just inform them to use `blueos.local` when using USB.

```suggestion
It's also possible to connect to the Raspberry through its USB-C connection. Once it's connected, the system should be available at `blueos.local`. 
```